sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -222,6 +224,78 @@ def recognize_entities(  # type: ignore
         except HttpResponseError as error:
             process_batch_error(error)
 
+    @distributed_trace
+    def recognize_pii_entities(  # type: ignore
+        self,
+        documents,  # type: Union[List[str], List[TextDocumentInput], List[Dict[str, str]]]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> List[Union[RecognizePiiEntitiesResult, DocumentError]]
+        """Recognize entities containing personal information for a batch of documents.
+
+        Returns a list of personal information entities ("SSN",
+        "Bank Account", etc) in the document.  For the list of supported entity types,
+        check https://aka.ms/tanerpii
+        for document length limits, maximum batch size, and supported text encoding.
+
+        See https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits
+        for document length limits, maximum batch size, and supported text encoding.
+
+        :param documents: The set of documents to process as part of this batch.
+            If you wish to specify the ID and language on a per-item basis you must
+            use as input a list[:class:`~azure.ai.textanalytics.TextDocumentInput`] or a list of
+            dict representations of :class:`~azure.ai.textanalytics.TextDocumentInput`, like
+            `{"id": "1", "language": "en", "text": "hello world"}`.
+        :type documents:
+            list[str] or list[~azure.ai.textanalytics.TextDocumentInput] or
+            list[dict[str, str]]
+        :keyword str language: The 2 letter ISO 639-1 representation of language for the
+            entire batch. For example, use "en" for English; "es" for Spanish etc.
+            If not set, uses "en" for English as default. Per-document language will
+            take precedence over whole batch language. See https://aka.ms/talangs for
+            supported languages in Text Analytics API.
+        :keyword str model_version: This value indicates which model will
+            be used for scoring, e.g. "latest", "2019-10-01". If a model-version
+            is not specified, the API will default to the latest, non-preview version.
+        :keyword bool show_stats: If set to true, response will contain document level statistics.
+        :return: The combined list of :class:`~azure.ai.textanalytics.RecognizePiiEntitiesResult`
+            and :class:`~azure.ai.textanalytics.DocumentError` in the order the original documents
+            were passed in.
+        :rtype: list[~azure.ai.textanalytics.RecognizePiiEntitiesResult,
+            ~azure.ai.textanalytics.DocumentError]
+        :raises ~azure.core.exceptions.HttpResponseError or TypeError or ValueError or NotImplementedError:
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/sample_recognize_pii_entities.py
+                :start-after: [START batch_recognize_pii_entities]
+                :end-before: [END batch_recognize_pii_entities]
```

Comment:
we can probably drop the batch_ prefix since i'm guessing this was copied from before we removed the single text methods

Reply:
I kept the batch prefix because all of the code snippets for all of the endpoints have it. I've filed [an issue](https://github.com/Azure/azure-sdk-for-python/issues/12688) to fix this for all languages and will fix this there instead. thanks!